guides/main_concepts/relevancy.md

```diff
@@ -126,6 +126,12 @@ To add your own ranking rule, you have to communicate either `asc` for ascending
 
 Add this rule to the existing list of ranking rules using the [settings route](/references/ranking_rules.md#update-ranking-rules).
 
+::: warning
+
+If you apply an ascending or descending sorting rule on an attribute, the values of the latter **must be numbers**. If any value is a string, it won't be converted into a numeric type.
```

Comment:
`it won't be converted into a numeric type.` What does this implies? This is confusing. 
` If any value is a string, it won't be converted into a numeric type.` If it is a string, but what if it is a boolean ? What if it is a null type? Do you mean `not a number` ?